das/cache_storage_to_das_adapter.go

```diff
@@ -88,8 +88,8 @@ func (s *emptyStorageService) Close(ctx context.Context) error {
 	return nil
 }
 
-func (s *emptyStorageService) ExpirationPolicy(ctx context.Context) ExpirationPolicy {
-	return DiscardAfterDataTimeout
+func (s *emptyStorageService) ExpirationPolicy(ctx context.Context) arbstate.ExpirationPolicy {
+	return arbstate.DiscardAfterDataTimeout
```

Comment:
Not sure what is the best value to return here. `DiscardAfterDataTimeout` is not an accurate description of this DAS's policy. Maybe we need a new policy type for this?

Reply:
Changed to DiscardImmediately.